{
  "cveId": "CVE-2024-27132",
  "title": "Insufficient sanitization in MLflow leads to XSS when running an untrusted recipe.",
  "description": "Insufficient sanitization in MLflow leads to XSS when running an untrusted recipe.\n\nThis issue leads to a client-side RCE when running an untrusted recipe in Jupyter Notebook.\n\nThe vulnerability stems from lack of sanitization over template variables.\n\n",
  "publishedDate": "2024-02-23T21:58:59.869000+00:00",
  "lastModifiedDate": "2024-08-14T15:28:19.332000+00:00",
  "severity": "HIGH",
  "cvssMetrics": [
    {
      "version": "3.1",
      "vectorString": "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:U/C:H/I:H/A:H",
      "baseScore": 7.5,
      "baseSeverity": "HIGH"
    }
  ],
  "epss": {
    "score": 0.0019,
    "percentile": 41.112
  },
  "riskScore": 37,
  "exploitationStatus": "UNKNOWN",
  "affectedSystems": {
    "cpeMatches": [],
    "vendors": [],
    "products": []
  },
  "references": [
    {
      "url": "https://research.jfrog.com/vulnerabilities/mlflow-untrusted-recipe-xss-jfsa-2024-000631930/",
      "source": "",
      "tags": []
    },
    {
      "url": "https://github.com/mlflow/mlflow/pull/10873",
      "source": "",
      "tags": []
    }
  ],
  "attackTechniques": [],
  "metadata": {
    "tags": ["CWE-79"],
    "requiresUserInteraction": null,
    "requiresPrivileges": null,
    "attackVector": null,
    "sources": [
      {
        "name": "CVEList",
        "url": "https://github.com/CVEProject/cvelistV5/blob/main/cves/2024/27xxx/CVE-2024-27132.json",
        "lastModified": "2024-08-14T15:28:19.332000+00:00"
      }
    ]
  }
}
